internal/compiler-bridge/src/main/scala/xsbt/Dependency.scala

```diff
@@ -484,7 +508,8 @@ final class Dependency(val global: CallbackGlobal) extends LocateClassFile with
 
       case m @ MacroExpansionOf(original) if inspectedOriginalTrees.add(original) =>
         // TODO: typesTouchedDuringMacroExpansion can be provided by compiler
-        // in the form of tree attachment
+        // in the form
+        // of tree attachment
```

Comment:
```suggestion
        // in the form of tree attachment
```

Minor change to avoid git blame on these lines points to #1507